src/Mvc/Directory.Build.props

```diff
@@ -0,0 +1,21 @@
+﻿<Project>
+  <Import
+    Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))\AspNetCoreSettings.props"
+    Condition=" '$(CI)' != 'true' AND '$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))' != '' " />
+
+  <Import Project="version.props" />
+  <Import Project="build\dependencies.props" />
+  <Import Project="build\sources.props" />
+
+  <PropertyGroup>
+    <Product>Microsoft ASP.NET Core MVC</Product>
+    <RepositoryUrl>https://github.com/aspnet/Mvc</RepositoryUrl>
```

Comment:
Can you update this value too? We should probably go back and cleanup this up for everything in 2.1 as well.